.github/workflows/ci_doctests.yaml

```
@@ -0,0 +1,113 @@
+# This workflow installs PyGMT and runs all doctests
+
+name: Doctests
+
+on:
+  # push:
+  #   branches: [ main ]
+  pull_request:
```

Comment this line before merging:
```suggestion
  # pull_request:
```